<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngKPropetryWidget</class>
 <widget class="QWidget" name="AngKPropetryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AngKPropetryWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bgWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="AngKTabProject" name="tabProject">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="AngKTabProgram" name="tabProgram">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
        <widget class="AngKTabSite" name="tabSite">
         <attribute name="title">
          <string>é¡µ</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AngKTabProject</class>
   <extends>QWidget</extends>
   <header>angktabproject.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AngKTabProgram</class>
   <extends>QWidget</extends>
   <header>angktabprogram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AngKTabSite</class>
   <extends>QWidget</extends>
   <header>angktabsite.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
